Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -523,6 +525,7 @@ ShaderCode GeneratePixelShaderCode(APIType ApiType, const pixel_shader_uid_data*
     else
     {
       out.Write("FRAGMENT_OUTPUT_LOCATION(0) out vec4 ocol0;\n");
+      out.Write("vec4 ocol1;\n");  // Consume the output we don't use
```

Comment:
This is a bit of a hack, but it's so much nicer than checking these conditions every time or passing a boolean everywhere.